### PR TITLE
Fixed various turtlebot issues

### PR DESCRIPTION
**Pull request by schmerl**

### CODE REVIEW

File: src/rosdiscover/interpreter/interpreter.py [hunk 1]
```diff
@@ -273,21 +274,31 @@ def _load(self,
 
         try:
             model = self.models.fetch(pkg, nodetype)
+            if isinstance(model, PlaceholderModel) and name != nodetype:
+                model = self.models.fetch(pkg, name)
```
Comment (ChrisTimperley) [inline src/rosdiscover/interpreter/interpreter.py:278]: might be worth adding a note about why `name` is used here

File: src/rosdiscover/models/amcl.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
-from ..interpreter import model
+# from ..interpreter import model
 
 M_PI = 3.14159265358979323846
 
 
-@model('amcl', 'amcl')
+# @model('amcl', 'amcl')
```
Comment (ChrisTimperley) [inline src/rosdiscover/models/amcl.py:7]: ?

File: src/rosdiscover/models/map_server.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
-from ..interpreter import model
+# from ..interpreter import model
 
 
-@model('map_server', 'map_server')
+# @model('map_server', 'map_server')
```
Comment (ChrisTimperley) [inline src/rosdiscover/models/map_server.py:4]: ?

File: src/rosdiscover/interpreter/interpreter.py (hunk 1 above)
Comment (schmerl) [reply src/rosdiscover/interpreter/interpreter.py:278]: Done

File: src/rosdiscover/models/amcl.py (hunk 2 above)
Comment (schmerl) [reply src/rosdiscover/models/amcl.py:7]: This is commented out so that we attempt to do recovery. Didn't want to delete in case we need it again.

Comment (schmerl) [reply src/rosdiscover/models/amcl.py:8]: Done, with comment indicating why.

File: src/rosdiscover/models/map_server.py (hunk 3 above)
Comment (schmerl) [reply src/rosdiscover/models/map_server.py:5]: Done with comments justifying why.

**Review by ChrisTimperley — APPROVED** [review]

**Pull request merged**